scripts/build.sh

```diff
@@ -919,6 +919,7 @@ clean_iso_dir()
 		return 0
 	fi
 	rm -rf ${ISODIR} >/dev/null 2>/dev/null
+	chflags -R noschg ${ISODIR}/tmp/* >/dev/null 2>/dev/null
```

Comment:
How is this any different thatn the line right below it? The -R should also descend into ${ISODIR}/tmp no?

Reply:
I've not look any deeper why isodir isnt enough but the fact is that tmp/iso/ still have lib libexec and friends there which cant be deleted due to ... ypchpass: Operation not permitted.